packages/forms-web-app/src/config.js

```diff
@@ -64,6 +64,6 @@ module.exports = {
     port: Number(process.env.PORT || 3000),
     sessionSecret: process.env.SESSION_KEY,
     useSecureSessionCookie: process.env.USE_SECURE_SESSION_COOKIES === 'true',
-    googleAnalyticsId: 'G-RTYZW789M0',
+    googleAnalyticsId: process.env.GOOGLE_ANALYTICS_ID || 'googleID',
```

Comment:
Remove `|| 'googleID`. If it's not there, the analytics code shouldn't be displayed